framework/audio/midi/internal/sequencer.cpp

```diff
@@ -221,7 +272,7 @@ uint64_t Sequencer::max_ticks(const std::vector<Track>& tracks) const
     return maxTicks;
 }
 
-bool Sequencer::channel_eot(const Channel& chan) const
+bool Sequencer::channelEOT(const Channel& chan) const

```

Comment:
what is the EOT?

Reply:
end of track - this is a common abbreviation for midi, used in many places